notebook/static/notebook/js/commandpalette.js

```diff
@@ -164,7 +164,11 @@ define(function(require){
         // now src is the right structure for typeahead
 
         input.typeahead({
-          emptyTemplate: "No results found for <pre>{{query}}</pre>",
+          emptyTemplate: function(query) {
+            return $('<div>').text("Nao results found for").append(
```

Comment:
No, rather than Nao (although I'd love if we just start saying "nah", "nao", "NOPE", etc in place of 'No' throughout the code)